rubin_sim/scheduler/detailers/detailer.py

```diff
@@ -169,15 +169,18 @@ def __call__(self, observation_list, conditions):
 class Filter_nexp(Base_detailer):
     """Demand one filter always be taken as a certain number of exposures
     """
```

Comment:
This looks fine, but I'm curious if it can be applied per-survey or per-area or something like that? For example, if 50s visits in the galactic bulge but 30s visits in the low-dust WFD region were desired.

Reply:
It would not be particularly straightforward unless you want to break different regions into different survey objects.

Reply:
Got it. And once they're separate survey objects, we have different issues combining them I think? As in, they compete differently than if they're just in one target map, and we would have edge effects where previously we did not? (along the boundaries of those regions, where previously they just were adjacent).

Reply:
The dithering would still work, so there shouldn't be huge edge effects. But it would mean an observation blob couldn't cross the boundaries anymore. The bulge may be small enough that things would start to break.